src/main/java/core/basesyntax/FigureList.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public enum FigureList {
```

Comment:
It is not a proper name for Enum) Enum is not a list 
```suggestion
public enum Figure {
```